ocp-templates/ibutsu-server-openshift.yaml

```diff
@@ -104,21 +69,31 @@ objects:
         - ibutsu-frontend
         from:
           kind: ImageStreamTag
-          name: ibutsu-frontend:latest
+          name: frontend:latest
           namespace: ${NAMESPACE}
       type: ImageChange
     - type: ConfigChange
 # -----------------------------------------------
 - kind: ImageStream
   apiVersion: v1
   metadata:
-    name: ibutsu-frontend
+    name: frontend
     annotations:
       description: "The frontend of Ibutsu server"
-      openshift.io/image.insecureRepository: "true"
   spec:
     lookupPolicy:
-      local: true
+      local: false
+    tags:
+    - annotations: null
+      from:
+        kind: DockerImage
+        name: quay.io/ibutsu/frontend
+      generation: 3
+      importPolicy:
+        scheduled: true
```

Comment:
Note: this will check for new images on quay every 15 minutes. So when releasing, a new deployment may not happen until after ~15 mins of a deploy on quay.

Reply:
I think 15 minutes is OK.